tests/test_utils.py

```diff
@@ -439,15 +452,20 @@ def test_quadratic_weighted_kappa_error():
 
 
 def test_partial_correlations_with_singular_matrix():
-
+    # This test is expected to pass UserWarning becaus
+    # of singularity
     expected = pd.DataFrame({0: [1.0, -1.0], 1: [-1.0, 1.0]})
     df_singular = pd.DataFrame(np.tile(np.random.randn(100), (2, 1))).T
-    assert_frame_equal(partial_correlations(df_singular), expected)
+    with warnings.catch_warnings(record=True) as w:
+        assert_frame_equal(partial_correlations(df_singular), expected)
+    eq_(len(w), 1)
+    assert issubclass(w[-1].category, UserWarning)
```

Comment:
```suggestion
    assert issubclass(warning_list[-1].category, UserWarning)
```

Same comment here re: `[0]` vs `[1]`